src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,263 @@
+
+class PpaCreationTestcase(sas_testcase.SasTestCase):
+  """
+  Implementation of PCR tests to verify the area of the non-overlapping
+  difference between the maximum PPA boundary created by SAS UUT shall be no more than
+  10% of the area of the maximum PPA boundary created by the Reference Model.
+  """
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_PCR_1_default_config(self, filename):
+    """ Generates the WinnForum configuration for PCR 1. """
+
+    # Load PAL records.
+    pal_record_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_1.json')))
+    pal_record_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'pal_record_2.json')))
+
+    # Set the values of fipsCode in pal_records_a and b to make them adjacent.
+    # 20063955100 and 20063955200 respectively.
+    pal_record_a['fipsCode'] = 20063955100
+    pal_record_b['fipsCode'] = 20063955200
+
+    # Set the PAL frequency.
+    pal_low_frequency = 3570000000
+    pal_high_frequency = 3580000000
+
+    # Load device info.
+    device_a = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_b = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Set the same user ID for all devices
+    device_b['userId'] = device_a['userId']
+
+    # Device_a is Category A.
+    self.assertEqual(device_a['cbsdCategory'], 'A')
+
+    # Device_b is Category B with conditionals pre-loaded.
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+
+    # Set the values of fipsCode in pal_records_a and b to make them adjacent.
+    # 20063955100 and 20063955200 respectively
+    pal_records = makePalRecordsConsistent([pal_record_a, pal_record_b],
+                                           pal_low_frequency, pal_high_frequency,
+                                           device_a['userId'])
+    
+    # Set the locations of devices to reside with in service area
+    device_a['installationParam']['latitude'], device_a['installationParam'][
+        'longitude'] = 39.0373, -100.4184
+    device_b['installationParam']['latitude'], device_b['installationParam'][
+        'longitude'] = 39.0378, -100.4785
+
+    # Set the AntennaGain and EIRP capability
+    device_a['installationParam']['eirpCapability'] = 30
+    device_b['installationParam']['eirpCapability'] = 47
+    device_a['installationParam']['antennaGain'] = 16
+    device_b['installationParam']['antennaGain'] = 16
+
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    conditionals = {'registrationData': [conditionals_b]}
```

Comment:
Please make this consistent with PR #479.

Reply:
We have   made changes to align with PR #479